Trimble.Modus.Components/Helpers/AppDataHelper.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Trimble.Modus.Components.Helpers
+{
+    public class AppDataHelper
+    {
+        public static T ParseEnum<T>(string value)
```

Comment:
Can you add comment to this method about what this do?

Reply:
Comments added